lib/web/apiserver.go

```diff
@@ -767,6 +767,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.POST("/webapi/roles", h.WithAuth(h.upsertRoleHandle))
 	h.PUT("/webapi/roles/:name", h.WithAuth(h.upsertRoleHandle))
 	h.DELETE("/webapi/roles/:name", h.WithAuth(h.deleteRole))
+	h.GET("/webapi/preset-roles", h.WithUnauthenticatedHighLimiter(h.getPresetRoles))
```

Comment:
I don't think we typically put - characters in URL paths.

Reply:
I'll rename it to `presetroles`